src/fidesops/task/graph_task.py

```diff
@@ -128,6 +128,15 @@ def grouped_fields(self) -> Set[Optional[str]]:
         """
         return self.traversal_node.node.collection.grouped_inputs or set()
 
+    @property
+    def dependent_identity_fields(self) -> bool:
+        """If the current collection needs inputs from other collections, in addition to its seed data."""
+        collection = self.traversal_node.node.collection
+        grouped_fields = [
+            collection.field(FieldPath(field)) for field in self.grouped_fields
+        ]
+        return any(field.identity for field in grouped_fields)
```

Comment:
We could save some iterations here using a for loop:

```
for field in grouped_fields:
    if collection.field(FieldPath(field)).identity:
        return True
```

then we also wouldn't need the overlap of `grouped_fields` and `self.grouped_fields`, where one is a different format.

Reply:
Good idea